sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -615,3 +626,80 @@ inline bfloat16 getBfloat16WithRoundingMode(const Ty &a) {
 } // namespace ext::oneapi
 } // namespace _V1
 } // namespace sycl
+
+// Specialization of some functions in namespace `std`.
+namespace std {
```

Comment:
Subjective, but I prefer

```
// in global ns
template <> struct std::{type}<types...> { ... };
```

Reply:
I don't have a strong preference, but the current style is what we use for `hash` in other headers, so I would like to keep it as is. Then we can change it at a global level if we want.